schema.yaml

```diff
@@ -80,6 +80,15 @@ properties:
     minItems: 1
     items: *n_y_items
 
+  upload_aws_image:
+    type: array
+    description: >-
+      Whether to upload the repository to AWS to build and host a Docker image of the project.
+      NOTE: requires CodeBuild configuration to be updated separately to have the image built on AWS.
+    uniqueItems: true
+    minItems: 1
+    items: *n_y_items
```

Comment:
I see some of the other y/n params have additional config like this, where the default and options are explicit:

```
      default: "n"
      enum: ["y", "n"]
```

Is there an implicit default here? Based on the unit tests, it seems to default to "no". Is that correct?

Reply:
`*n_y_items` is a YAML anchor call to `&n_y_items`, in which those values are defined.